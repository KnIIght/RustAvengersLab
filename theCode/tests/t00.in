1
Stan 
luck 
none
