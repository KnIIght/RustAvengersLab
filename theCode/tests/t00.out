My name is Stan, my power is luck.
With great power, comes great responsibility.
Stan is always worthy...
